tests/test_reporting.py

```diff
@@ -106,10 +107,16 @@ def test_generate_report_dict_script(self):
 
         reporting.generate_report(args)
 
-        pygeoprocessing.testing.assert_text_equal(
-            args['out_uri'],
-            os.path.join(
-                REGRESSION_DATA, 'html_reports', 'report_dict_script.html'))
+        regression_path = os.path.join(
+            REGRESSION_DATA, 'html_reports', 'report_dict_script.html')
+        for source_line, regression_line in itertools.zip_longest(

```

Comment:
This is fun and a creative solution!

Reply:
Gotta make sure we don't ignore any lines!